src/models/unified_api_server.py

```
@@ -0,0 +1,443 @@
+        ):
+            """Complete pipeline: Audio -> Transcription -> Summary -> Emotion Analysis."""
+            pipeline_start = time.time()
+            pipeline_steps = []
+
+            try:
+                # Step 1: Transcribe audio
+                pipeline_steps.append("transcription")
+                if not self.models["transcriber"]:
+                    raise HTTPException(status_code=503, detail="Transcription model not available")
+
+                temp_path = f"/tmp/{file.filename}"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)